### PR TITLE
Further clarify OCR env vars

### PR DESCRIPTION
**Pull request by connorwstein**
Deduplicate handling of global OCR env vars and improve readability

### CODE REVIEW

File: core/config/general_config.go [hunk 1]
```diff
@@ -200,11 +197,12 @@ type GlobalConfig interface {
 	GlobalMinIncomingConfirmations() (uint32, bool)
 	GlobalMinRequiredOutgoingConfirmations() (uint64, bool)
 	GlobalMinimumContractPayment() (*assets.Link, bool)
-	GlobalOCRContractConfirmations() (uint16, bool)
-	GlobalOCRContractTransmitterTransmitTimeout() (time.Duration, bool)
-	GlobalOCRDatabaseTimeout() (time.Duration, bool)
-	GlobalOCRObservationGracePeriod() (time.Duration, bool)
-	GlobalOCR2ContractConfirmations() (uint16, bool)
+
+	OCR1Config
```
Comment (connorwstein) [inline core/config/general_config.go:201]: Keeping this more fine grained will help reusability in the ocr packages

File: core/config/general_config.go [hunk 2]
```diff
@@ -1318,41 +1316,6 @@ func (*generalConfig) GlobalMinimumContractPayment() (*assets.Link, bool) {
 	}
 	return val.(*assets.Link), ok
 }
-func (*generalConfig) GlobalOCRContractTransmitterTransmitTimeout() (time.Duration, bool) {
```
Comment (connorwstein) [inline core/config/general_config.go:1321]: These were actually duplicated with the ones in ocr_config after a develop merge

File: core/config/schema.go [hunk 3]
```diff
@@ -170,20 +172,23 @@ type ConfigSchema struct {
 	OCR2MonitoringEndpoint                 string        `env:"OCR2_MONITORING_ENDPOINT"`
 
 	// OCR V1
-	OCRBlockchainTimeout                  time.Duration `env:"OCR_BLOCKCHAIN_TIMEOUT" default:"20s"`
+	// Per-chain defaults
```
Comment (connorwstein) [inline core/config/schema.go:175]: Making it clear that the ones which don't have all-chain defaults here have per-chain defaults

**Review by samsondav — DISMISSED** [review]
This is making the config object much cleaner, nice work

**Review by jmank88 — APPROVED** [review]

**Review by jmank88 — APPROVED** [review]

**Pull request merged**